docs/release-notes.md

```diff
@@ -10,6 +10,161 @@ toc_max_heading_level: 2
+- New [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page styling. See [Managing findings in Semgrep App](https://semgrep.dev/docs/semgrep-app/findings/) documentation for additional information.
+- Semgrep App experience is generally improved due to a significant number of fixed bugs.
+
+### Semgrep CLI
+
+These release notes include upgrades for versions ranging between 0.116.0 and 0.118.0.
+
+#### Additions
+
+##### Taint mode
+
+- Taint mode now tracks taint coming from the default values of function parameters. For example, given `def test(url = "http://example.com"):`, if `"http://example.com"` is a taint source (as a consequence of not using TLS protocol), then `url` is marked as tainted. (Issue [#6298](https://github.com/returntocorp/semgrep/issues/6298))
+- Two new rule options that help to minimize false positives:
+    - The`taint_assume_safe_indexes`, which makes Semgrep assume that an array-access expression is safe even if the index expression is tainted. Otherwise Semgrep assumes that for example: `a[i]` is tainted if `i` is tainted, even if `a` is not. Enabling this option is recommended for high-signal rules, whereas disabling is preferred for audit rules. Currently, it is disabled by default to attain backwards compatibility, but this can change in the near future after some evaluation. To enable this option, include the `taint_assume_safe_indexes: true` under the `options` key. For example:
+        ```yaml
+        rules:
+      - id: tainted
+        languages:
+          - python
+        message: Match
+        mode: taint
+        options:
+          taint_assume_safe_indexes: true
+        pattern-sinks:
+          - pattern: sink(...)
+        pattern-sources:
+          - pattern: tainted(...)
+        severity: WARNING
+        ```
+
+    - The `taint_assume_safe_functions`, makes Semgrep assume that function calls do **not** propagate taint from their arguments to their output. Otherwise, Semgrep always assumes that functions may propagate taint. This is intended to replace **not-conflicting** sanitizers (added in v0.69.0, for more information, see [Minimizing false positives via sanitizers](https://semgrep.dev/docs/writing-rules/data-flow/taint-mode/#minimizing-false-positives-via-sanitizers)) in the future. This option is still experimental and needs to be complemented by other changes in future releases. To enable this option, include the `taint_assume_safe_functions: true` under the `options` key. For example:
+        ```yaml    
+        rules:
+          - id: tainted
+            languages:
+              - php
+                message: Match
+            mode: taint
+            options:
+              taint_assume_safe_functions: true
+            pattern-sinks:
+              - pattern: sink(...)
+            pattern-sources:
+              - pattern: tainted(...)
+            severity: WARNING
+        ```
```

Comment:
@pabloest we mentioned both of these under https://semgrep.dev/docs/writing-rules/rule-syntax/#options 
But sometimes documenting how to do something in release notes can also be beneficial. If you do not find it helpful let me know and I will remove the code snippets in the release notes and link to rule-syntax section linked above.

![Screenshot 2022-10-31 at 17 13 20](https://user-images.githubusercontent.com/26669307/199055949-e6c855a8-ee28-4288-88db-32e480d6dad1.png)

Reply:
I think if they will be not changing over time (I expect they won’t), then this looks good to me.

Reply:
They may change, they are experimental...

Reply:
But I may remove them from release notes once they will be updated.